### PR TITLE
chore(curriculum): add colored markers to new cert

### LINKED ISSUE #592: chore: add colored markers workshop to main repo
We are using the existing colored markers project found in the Responsive Web Design cert and adding that to the new frontend cert.
We are keeping the ids the same as the original ids from the RWD cert project.

**NOTE**
Please review the sheet for the lecture scripts. The lectures come before the workshops and some of the step descriptions will need to be updated. 

For example, if a lecture script already introduces a concept, then the language in the step shouldn't say "Now you are going to learn about X". It doesn't make sense to continuously repeat introductions, if it was already introduced just before this. 

You can update those steps to say "In the lecture videos, you learn how to work with X like this: [show example]" .
That makes sense.

At the end of the day, the flow of the curriculum needs to make sense to campers.  So we need to make sure the flow between the lecture videos, workshops and labs all make sense. 👍

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes https://github.com/freeCodeCamp/CurriculumExpansion/issues/592

### CODE REVIEW

**Review by zairahira — APPROVED** [review]

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -1901,7 +1901,12 @@
           "Test what you've learned in this quiz of pseudo-classes in CSS."
         ]
       },
-      "fvcj": { "title": "63", "intro": [] },
+      "workshop-colored-markers": {
+        "title": "Create a Set of Colored Markers",
+        "intro": [
+          "In this course, you'll build a set of colored markers. You'll practice different ways to set color values and how to pair colors with each other."
```
Comment (Dario-DC) [inline client/i18n/locales/english/intro.json:1907]:
```suggestion
          "In this workshop, you'll build a set of colored markers. You'll practice different ways to set color values and how to pair colors with each other."
```

File: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/61696ef7ac756c829f9e4048.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+id: 61696ef7ac756c829f9e4048
+title: Step 2
+challengeType: 0
+dashedName: step-2
+---
+
+# --description--
+
+In this project you'll work with an external CSS file to style the page. We've already created a `styles.css` file for you. But before you can use it, you'll need to link it to the page.
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/workshop-colored-markers/61696ef7ac756c829f9e4048.md:10]:
```suggestion
In this project you'll work with an external CSS file to style the page. A `styles.css` file has been already created for you. But before you can use it, you'll need to link it to the page.
```

File: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/617bbb6b97a83f6d1f7d6e38.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+---
+id: 617bbb6b97a83f6d1f7d6e38
+title: Step 26
+challengeType: 0
+dashedName: step-26
+---
+
+# --description--
+
+Now that you''ve practiced with secondary colors, you'll review how to create <dfn>tertiary colors</dfn>. Tertiary colors are created by combining a primary with a nearby secondary color.
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/workshop-colored-markers/617bbb6b97a83f6d1f7d6e38.md:10]:
```suggestion
Now that you've practiced with secondary colors, you'll review how to create <dfn>tertiary colors</dfn>. Tertiary colors are created by combining a primary with a nearby secondary color.
```

File: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/61b0936d9e7fc020b43b1bb8.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
+---
+id: 61b0936d9e7fc020b43b1bb8
+title: Step 73
+challengeType: 0
+dashedName: step-73
+---
+
+# --description--
+
+It looks like your sleeve disappeared, but don't worry -- it's still there. What happened is that your new cap `div` is taking up the entire width of the marker, and is pushing the sleeve down to the next line.
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/workshop-colored-markers/61b0936d9e7fc020b43b1bb8.md:10]:
```suggestion
It looks like your sleeve disappeared, but don't worry — it's still there. What happened is that your new cap `div` is taking up the entire width of the marker, and is pushing the sleeve down to the next line.
```

File: client/i18n/locales/english/intro.json [hunk 5]
```diff
@@ -1901,7 +1901,12 @@
           "Test what you've learned in this quiz of pseudo-classes in CSS."
         ]
       },
-      "fvcj": { "title": "63", "intro": [] },
+      "workshop-colored-markers": {
+        "title": "Create a Set of Colored Markers",
```
Comment (Dario-DC) [inline client/i18n/locales/english/intro.json:1905]:
```suggestion
        "title": "Build a Set of Colored Markers",
```

File: client/src/pages/learn/front-end-development/workshop-colored-markers/index.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+title: Introduction to the Create a Set of Colored Markers
```
Comment (Dario-DC) [inline client/src/pages/learn/front-end-development/workshop-colored-markers/index.md:2]:
```suggestion
title: Introduction to the Build a Set of Colored Markers
```

File: client/src/pages/learn/front-end-development/workshop-colored-markers/index.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+block: workshop-colored-markers
+superBlock: front-end-development
+---
+
+## Introduction to the Create a Set of Colored Markers
```
Comment (Dario-DC) [inline client/src/pages/learn/front-end-development/workshop-colored-markers/index.md:7]:
```suggestion
## Introduction to the Build a Set of Colored Markers
```

File: curriculum/challenges/_meta/workshop-colored-markers/meta.json [hunk 8]
```diff
@@ -0,0 +1,370 @@
+{
+  "name": "Create a Set of Colored Markers",
```
Comment (Dario-DC) [inline curriculum/challenges/_meta/workshop-colored-markers/meta.json:2]:
```suggestion
  "name": "Build a Set of Colored Markers",
```

**Review by Dario-DC — COMMENTED** [review]
I would stick to "build" but I don't have a strong opinion on this.

File: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/616d3a67ccf800ad94ec89ae.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
+---
+id: 616d3a67ccf800ad94ec89ae
+title: Step 4
+challengeType: 0
+dashedName: step-4
+---
+
+# --description--
+
+Now you'll add some elements that you'll eventually style into color markers.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-colored-markers/616d3a67ccf800ad94ec89ae.md:10]:
```suggestion
Now, you'll add some elements that you'll eventually style as color markers.
```

File: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/616d3a67ccf800ad94ec89ae.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+First, within the `body` element, add a `div` element and set its `class` attribute to `container`. Make sure the `div` element is below the `h1` element.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-colored-markers/616d3a67ccf800ad94ec89ae.md:12]:
```suggestion
First, within the `body` element, add a `div` element with a `class` attribute set to `container`. Make sure the `div` element is placed below the `h1` element.
```

File: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/616d525007b8c5d8b3308b1c.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
+---
+id: 616d525007b8c5d8b3308b1c
+title: Step 8
+challengeType: 0
+dashedName: step-8
+---
+
+# --description--
+
+Your marker would look better if it was centered on the page. An easy way to do that is with the `margin` <dfn>shorthand property</dfn>.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-colored-markers/616d525007b8c5d8b3308b1c.md:10]:
```suggestion
Your marker would look better if it were centered on the page. An easy way to do this is by using the `margin` <dfn>shorthand property</dfn>.
```

File: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/61764c602bee6974e7790f35.md [hunk 12]
```diff
@@ -0,0 +1,79 @@
+---
+id: 61764c602bee6974e7790f35
+title: Step 11
+challengeType: 0
+dashedName: step-11
+---
+
+# --description--
+
+To give the markers different colors, you will need to add a unique class to each one. Multiple classes can be added to an element by listing them in the `class` attribute and separating them with a space. For example, the following adds both the `animal` and `dog` classes to a `div` element.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-colored-markers/61764c602bee6974e7790f35.md:10]:
```suggestion
To give the markers different colors, you’ll need to add a unique class to each one. You can add multiple classes to an element by listing them in the `class` attribute and separating them with a space. For example, the following code adds both the `animal` and `dog` classes to a `div` element:
```

File: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/61764fdda535587e1fefb3aa.md [hunk 13]
```diff
@@ -0,0 +1,88 @@
+---
+id: 61764fdda535587e1fefb3aa
+title: Step 14
+challengeType: 0
+dashedName: step-14
+---
+
+# --description--
+
+Add the class `two` to the second marker `div`, and add the class `three` to the third marker `div`.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-colored-markers/61764fdda535587e1fefb3aa.md:10]:
```suggestion
Add the class `two` to the second marker `div` and the class `three` to the third marker `div`.
```

File: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/617b65579ce424bf5f02ca73.md [hunk 14]
```diff
@@ -0,0 +1,84 @@
+---
+id: 617b65579ce424bf5f02ca73
+title: Step 16
+challengeType: 0
+dashedName: step-16
+---
+
+# --description--
+
+As you have learned in the lectures, there are two main color models: the <dfn>additive</dfn> RGB (red, green, blue) model used in electronic devices, and the <dfn>subtractive</dfn> CMYK (cyan, magenta, yellow, black) model used in print.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-colored-markers/617b65579ce424bf5f02ca73.md:10]:
```suggestion
As you have learned in the lectures, there are two main color models: the <dfn>additive</dfn> RGB (red, green, blue) model, used in electronic devices, and the <dfn>subtractive</dfn> CMYK (cyan, magenta, yellow, black) model, used in print.
```

File: curriculum/challenges/english/25-front-end-development/workshop-colored-markers/617b65579ce424bf5f02ca73.md [hunk 15]
```diff
@@ -0,0 +1,84 @@
...
+
+In this project, you'll work with the RGB model. This means that colors begin as black, and change as different levels of red, green, and blue are introduced. An easy way to see this is with the CSS `rgb` function.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-colored-markers/617b65579ce424bf5f02ca73.md:12]:
```suggestion
In this project, you'll work with the RGB model. This means colors start as black and change as different levels of red, green, and blue are introduced. An easy way to see this is with the CSS `rgb` function.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**